README.md

```diff
@@ -28,19 +28,27 @@ $ npm install --save dual-quat-to-mat4
 ```js
 var dualQuatToMat4 = require('dual-quat-to-mat4')
 
-var dualQuat = dualQuatToMat4([0, 0, 0, 1, 0, 0, 0, 0])
+var dualQuat = dualQuatToMat4([], [0, 0, 0, 1, 0, 0, 0, 0])
```

Comment:
Might just replace this line with:

```js
 var dualQuatToMat4 = require('dual-quat-to-mat4')

var matrix = []
dualQuatToMat4(matrix, [0, 0, 0, 1, 0, 0, 0, 0])

console.log(matrix)
// [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1]
```

In order to illustrate the recommended pre-allocated usage